analytic_engine/src/row_iter/merge.rs

```diff
@@ -619,6 +620,8 @@ pub struct MergeIterator {
     schema: RecordSchemaWithKey,
     record_batch_builder: RecordBatchWithKeyBuilder,
     origin_streams: Vec<SequencedRecordBatchStream>,
+    /// ssts are kept here to avoid them from being purged.
+    _ssts: Vec<Vec<FileHandle>>,
```

Comment:
Maybe we can remove the `_` prefix by adding `#[allow(dead_code)]`.

Reply:
The directive will be added for whole struct, not this field, which is a bad practice, as others don't know why dead_code is required for this struct.

Reply:
I guess the directive can be applied only to a specific field.

Reply:
Ooops, I will fix this.